ui_advanced_config.py

```diff
@@ -256,7 +326,56 @@ def BuildUI(self):
             padx=10,
             pady=10,
         )
+        self.objectsCreateButton.grid(
+            row=13,
+            column=1,
+            columnspan=2,
+            sticky="nsew",
+            ipadx=2,
+            ipady=2,
+            padx=10,
+            pady=10,
+        )
+        self.objectsDeleteButton.grid(
+            row=14,
+            column=1,
+            columnspan=2,
+            sticky="nsew",
+            ipadx=2,
+            ipady=2,
+            padx=10,
+            pady=10,
+        )
 
+        # Make Map Widgets
+        self.selectMapsCombo = uiComboBox(master=self.mapsColumn)
+        self.mapsLabel = uiLabel(master=self.mapsColumn, text="Maps")
+        self.mapsUpdateButton = uiButton(
+            master=self.mapsColumn, command=self.updateMapsJSON, text="Update"
+        )
```

Comment:
Updating maps without changing values has changed width and height to string values. Picture attached.

![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/97774046/d7778f4f-74b0-447b-9fb4-5dbb58adc000)